src/api/docker-compose.yml

```diff
@@ -46,3 +46,48 @@ services:
       - 'traefik.http.middlewares.strip_api_prefix.stripprefix.prefixes=/${API_VERSION}/image'
       - 'traefik.http.middlewares.strip_api_prefix.stripprefix.forceSlash=true'
       - 'traefik.http.routers.image.middlewares=strip_api_prefix'
+
+  redis:
+    image: redis:latest
+    ports:
+      - ${REDIS_PORT}:${REDIS_PORT}
+    command: ['redis-server', '--appendonly', 'yes']
+    volumes:
+      - ../../../redis-data:/data
```

Comment:
I wonder where we should put volumes related to the microservices.  The root is one option for sure, but maybe we should have a dedicated directory where we can find these things.  cc @manekenpix, @raygervais.

Reply:
@humphd this is how we currently do it on prod and dev:

[Directory Structure](https://github.com/Seneca-CDOT/telescope/blob/master/docs/staging-production-deployment.md#directory-structure)